src/webgpu/api/validation/copyBetweenLinearDataAndTexture_writeTextureSpecific.spec.ts

```diff
@@ -0,0 +1,126 @@
+export const description = `
+writeTexture validation tests.
+`;
+
+import { params, poptions, pbool } from '../../../common/framework/params_builder.js';
+import { makeTestGroup } from '../../../common/framework/test_group.js';
+import { kTextureFormats, kTextureFormatInfo } from '../../capability_info.js';
+
+import { CopyBetweenLinearDataAndTextureTest, TestMethod } from './copyBetweenLinearDataAndTexture.js';
+
+export const g = makeTestGroup(CopyBetweenLinearDataAndTextureTest);
+
+g.test('bound_on_bytes_per_row')
+  .params(
+    params()
+      .combine([
+        { blocksPerRow: 2, additionalBytesPerRow: 0, copyWidthInBlocks: 2 }, // always success
+        { blocksPerRow: 2, additionalBytesPerRow: 3, copyWidthInBlocks: 3 }, // success if bytesPerBlock <= 3
+        { blocksPerRow: 2, additionalBytesPerRow: 5, copyWidthInBlocks: 3 }, // success if bytesPerBlock <= 5
+        { blocksPerRow: 1, additionalBytesPerRow: 0, copyWidthInBlocks: 0 }, // always failure
```

Comment:
Shouldn't this one and the one below succeed because blockPerRow >= copyWidthInBlocks?